src/jdk.httpserver/share/classes/com/sun/net/httpserver/Headers.java

```diff
@@ -36,43 +41,48 @@
  * the value associated with each key is
  * a {@link List}{@literal <}{@link String}{@literal >} with one
  * element for each occurrence of the header name in the request or response.
- * <p>
- * For example, if a response header instance contains
+ *
+ * <p> For example, if a response header instance contains
  * one key "HeaderName" with two values "value1 and value2"
  * then this object is output as two header lines:
+ *
  * <blockquote><pre>
  * HeaderName: value1
  * HeaderName: value2
  * </pre></blockquote>
- * <p>
- * All the normal {@link java.util.Map} methods are provided, but the following
+ *
+ * <p> All the normal {@link java.util.Map} methods are provided, but the following
  * additional convenience methods are most likely to be used:
+ *
  * <ul>
- * <li>{@link #getFirst(String)} returns a single valued header or the first value of
- * a multi-valued header.</li>
- * <li>{@link #add(String,String)} adds the given header value to the list for the given key</li>
- * <li>{@link #set(String,String)} sets the given header field to the single value given
- * overwriting any existing values in the value list.
- * </ul><p>
- * All methods in this class accept <code>null</code> values for keys and values. However, null
+ *     <li>{@link #getFirst(String)} returns a single valued header or the first value of
+ *     a multi-valued header.
+ *     <li>{@link #add(String,String)} adds the given header value to the list for the given key
+ *     <li>{@link #set(String,String)} sets the given header field to the single value given
+ *     overwriting any existing values in the value list
```

Comment:
For consistency, either:
  - no item should end with a punctuation sign
  - or all of them should end with a coma, except the last that should end with a full stop.
  - or all of them should end with a full stop.

I am not sure what is the best, but it is weird that the first item ends with a full stop and the other two don't.

Reply:
Change made in commit 4de7e6c - all lines now end with a full stop